CHANGELOG.md

```diff
@@ -5,7 +5,90 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 
-## [1.6.0] - 2022-MM-DD
+## [1.7.0] - 2022-MM-DD
```

Comment:
Adding this already so PRs for latter milestones can update already without conflicts when we remove spaces and cleanup the 1.6.0 changelog

Reply:
```suggestion
## [UnReleased] - 2022-MM-DD
```

Reply:
@carmocca

Reply:
For some reason, I cannot commit the suggestion